src/apprt/gtk/Surface.zig

```diff
@@ -2359,6 +2359,9 @@ pub fn defaultTermioEnv(self: *Surface) !std.process.EnvMap {
         env.remove("GDK_PIXBUF_MODULEDIR");
         env.remove("GDK_PIXBUF_MODULE_FILE");
         env.remove("GTK_PATH");
+        env.remove("XDG_CONFIG_HOME");
+        env.remove("XDG_CACHE_HOME");
+        env.remove("XDG_DATA_HOME");
```

Comment:
I'm not sure this is safe. User or system could set this via systemd and wouldn't the launcher script inherit those paths?

Reply:
__

> I'm not sure this is safe. User or system could set this via systemd and wouldn't the launcher script inherit those paths?

I think this should be fine. If a user is setting them in their own environment, they will get set when their shell starts. It's just environment variables that are set log enough to get ghostty running, before their shell starts.

Reply:
I don't set these variables in my shell init script. I set them in systemd.

Reply:
I'm also not sure about this, because the user may not be launching a shell, they can launch any `command` that could rely on these env vars, no?

Reply:
This is the blocker for this PR, any feedback?